Dailymotion.php

```diff
@@ -487,11 +506,11 @@ public function call($resource, $args = array())
                 throw new DailymotionApiException($message, $code);
             }
         }
-        elseif (!isset($result['result']))
+        if (!$isGetRequest && !isset($result['result']))
```

Comment:
return the result when its get request

```suggestion
        if ($isGetRequest && !isset($result['result'])) { return $result; }
       
        if (!isset($result['result']))
```